docs/usage/schema.md

```diff
@@ -190,10 +202,10 @@ print(schema_json_of(Pet, title='The Pet Schema', indent=2))
       "required": [
         "pet_type",
         "cat_name"
-      ]
+      ],
+      "title": "Cat"
```

Comment:
This seems like it just moved from above? Which makes sense since (1) we may be inserting things into the dict in different order and (2) the examples are in strings so if `print()` is different they'll be "updated"